index.html

```diff
@@ -130,8 +145,12 @@ <h2> Tonic</h2>
             <h2> Multi-Post Stories</h2>
```

Comment:
- [ ] Good job on your design 👍🏽 . Kindly ensure that all your project card titles match the Figma design. Please increase font-size and font-weight 😄

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/199579348-657700dc-9390-4a5e-8345-8af92ac0ee1c.png))  | ![image](https://user-images.githubusercontent.com/63473584/199579385-550fb007-0f4f-46e1-913c-537a36b391b5.png)  |